Workbooks/Workloads/Custom Telegraf/Create new profile/settings.json

```diff
@@ -3,6 +3,6 @@
     "name":"Create new profile",
     "author": "Microsoft",
     "galleries": [
-        { "type": "workload-onboarding", "resourceType": "Azure Monitor", "order": 400 }
```

Comment:
should also verify if these all shows up in the gallery the way you expect

Reply:
changed to just an empty gallery array, so it won't show